src/gui/sdlmain.cpp

```diff
@@ -4565,7 +4565,7 @@ void restart_program(std::vector<std::string> & parameters) {
 	// nullptr for default
 	const LPSECURITY_ATTRIBUTES thread_attributes = nullptr;
 
-	const WINBOOL inherit_handles = FALSE;
+	const BOOL inherit_handles = FALSE;
```

Comment:
I just merged a fix for this a couple hours ago so might be a conflict when you rebase.